### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by tahseenzahra**
This pr includes,
- Bookstore design in [Zeplin](https://zeplin.io/).
- Add any missing JSX elements to your projects (without adding extra functionality)

### CODE REVIEW

File: src/css-modules/Navbar.module.css [hunk 1]
```diff
@@ -1,13 +1,49 @@
...
+.navbar {
+  background-color: #fff;
+}
+
+.navContainer {
+  margin: 0 6.25rem;
+  display: flex;
+  gap: 2.938rem;
+  align-items: center;
+}
```
Comment (zieeco) [inline src/css-modules/Navbar.module.css]: - Hey, great job with the styling. Please let's kindly improve the padding to add more space to make our design close to the template as much as we can.
- Let's also  add the missing icon to match the design as much as we can below:

*your design*
Screenshot from 2022-11-19 23-13-11

*template*
Screenshot from 2022-11-19 23-12-28

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @tahseenzahra ,


Great work job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/css-modules/Navbar.module.css (hunk 1 above)
Comment (tahseenzahra) [reply src/css-modules/Navbar.module.css]: Hi @zieeco thanks for giving a review i tried to give icon but its not working and I can't figure out the reason can you help me out in that

Comment (tahseenzahra) [reply src/css-modules/Navbar.module.css]: required
done

File: src/components/Navbar.js [hunk 2]
```diff
@@ -10,7 +10,11 @@ function Navbar() {
         <div className={NavbarCSS.logo}>Bookstore CMS</div>
         <Link className={NavbarCSS.navLink} to="/">Books</Link>
         <Link className={NavbarCSS.navLink} to="/categories">Categories</Link>
-        <span className="material-symbols-rounded user-icon" />
+        <div className="avatar">
+          <div className="user-icon">
+            <span />
+          </div>
+        </div>
```
Comment (zieeco) [inline src/components/Navbar.js:17]: - Great adding the icon, however, you've not aligned the icon correctly which should be at the extreme end of the right, and also, you've not used the correct color on the icon. I understand you've put in more effort. please kindly update this issue pointed out. I'd approve your project. Kindly keep in mind that being attentive to detail is a skill that would make you standout in the job market, let's improve on this skill :pray:

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @tahseenzahra  


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
_check the comment under the review_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navbar.js (hunk 2 above)
Comment (tahseenzahra) [reply src/components/Navbar.js:17]: thanks for your valuable comment sure i will be doing it

**Pull request merged**